generators/angular/generator.ts

```diff
@@ -48,10 +47,15 @@ import {
   isTranslatedAngularFile,
   translateAngularFilesTransform,
 } from './support/index.js';
+import type { AngularApplication } from './types.js';
 
 const { ANGULAR } = clientFrameworkTypes;
 
-export default class AngularGenerator extends BaseApplicationGenerator {
+export default class AngularGenerator extends BaseApplicationGenerator<
+  Entity,
+  AngularApplication,
+  DefaultTaskTypes<Entity, AngularApplication>
+> {
```

Comment:
- we are not ready to modularize Types there are too many common types.
- almost every blueprint uses BaseApplicationGenerator as base Angular blueprints included https://github.com/jhipster/generator-jhipster-micronaut/blob/257e7494606c9e45d103c1ee5e2642cc3e023544/generators/angular/generator.js#L1.
- we should try to inject the command definition to build the specific generator type in the future

Reply:
Should be ok for retrocompatibility now.
Quick question: do you know when V9 will begin?